example/android/app/src/main/AndroidManifest.xml

```diff
@@ -5,7 +5,17 @@
          flutter needs it to communicate with the running application
          to allow setting breakpoints, to provide hot reload, etc.
     -->
-    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+    <uses-permission android:name="android.permission.CHANGE_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
+    <uses-permission android:name="android.permission.WRITE_SETTINGS" />
+
+    <permission android:name="android.permission.OVERRIDE_WIFI_CONFIG" android:protectionLevel="signature|privileged" />
+    <uses-feature android:name="android.hardware.wifi" />
```

Comment:
does user need to add this in their project? should it be put in README?

Reply:
Yes, I'll put this on README

Reply:
I assume not all are required for basic operation, should we categorize them in say - basic (always required, this should be added to plugin itself, so that the user does not need to add them), used for WiFi only, used for AP only, or categories like that. Even explaining why additional permission is required could help user to pick and place which once to put in.

Reply:
I got it, that would be even clearer on what permission to use for certain uses.

Reply:
I'll update the docs

Reply:
@daadu Please check the updated readme